NuRadioReco/modules/channelGenericNoiseAdder.py

```diff
@@ -145,6 +160,27 @@ def bandlimited_noise(self, min_freq, max_freq, n_samples, sampling_rate, amplit
         elif type == 'rayleigh':
             fsigma = amplitude * sigscale / np.sqrt(2.)
             ampl[selection] = self.__random_generator.rayleigh(fsigma, nbinsactive)
+        elif type == "data-driven":
+            self.logger.warning("You have selected to generate data-driven noise NOTE THAT THE HARDWARE RESPONSE IS ALREADY INLUDED IN THIS NOISE and hence this module should be run AFTER incorporating hardware response.")
```

Comment:
I agree, releasing this warning all the time is not a suitable way. There are better ways! Can you remove this?